facia-press/app/frontpress/FapiFrontPress.scala

```diff
@@ -117,6 +107,49 @@ trait FapiFrontPress extends Logging with ExecutionContexts {
       .showReferences(QueryDefaults.references)
       .showAtoms("media")
 
+  def pressByPathId(path: String): Future[Unit] = {
+
+    val stopWatch: StopWatch = new StopWatch
+
+    val pressFuture = getPressedFrontForPath(path)
+      .map { pressedFront =>
+        val json: String = Json.stringify(Json.toJson(pressedFront))
+        FaciaPressMetrics.FrontPressContentSize.recordSample(json.getBytes.length, new DateTime())
+        putPressedJson(path, json)
+      }
+      .asFuture
+      .map(
+        _.fold(
+          e => {
+            StatusNotification.notifyFailedJob(path, isLive = isLiveContent, e)
+            throw new RuntimeException(s"${e.cause} ${e.message}")
+          },
+          _ => StatusNotification.notifyCompleteJob(path, isLive = isLiveContent))
+      )
+
+
+    pressFuture.onComplete {
+      case Success(_) =>
+        log.info(s"Successfully pressed $path in ${stopWatch.elapsed} ms")
```

Comment:
Is it worth putting `stopWatch.elapsed` in a `val` so we guarantee all the times here are consistent.